packages/gatsby-page-utils/src/watch-directory.ts

```diff
@@ -1,9 +1,13 @@
-const Promise = require(`bluebird`)
```

Comment:
Dropping bluebird could be a non-trivial change. Did you actually check whether this holds or was this more of a cleanup thing? While I'm in favor of removing bluebird where possible, in favor of native promises, these changes should not be part of the TS migration.

Please undo unless you can guarantee the change is trivial.